tests/test_views.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from django.test import RequestFactory
+
+from rest_framework_json_api import serializers, views
+from rest_framework_json_api.relations import ResourceRelatedField
+from rest_framework_json_api.utils import format_value
+
+from .models import BasicModel
+
+related_model_field_name = "related_field_model"
+
+
+@pytest.mark.urls("tests.urls")
```

Comment:
I think this line can be removed as you are using a request factory below and are not relying on urls in the test code.